todoapp/src/components/Taskdetails.tsx

```diff
@@ -0,0 +1,49 @@
+import useFetch from "./useFetch";
+import { useParams, useNavigate } from "react-router";
+import { Link } from "react-router-dom";
+
+const Taskdetails = () => {
+  const { id } = useParams();
+  const navigate = useNavigate();
+  const {error,isPending,data: todoItem} : any = useFetch("http://localhost:5000/todos/" + id);
+  console.log(todoItem);
+
+  const handleClick = () => {
+    fetch("http://localhost:5000/todos/" + id, {
```

Comment:
Try to create a wrapper or for now a useFetch custom hook

Reply:
Resolved !!